oomact_ros/cfg/kitti.info

```diff
@@ -0,0 +1,97 @@
+model {
+  Gravity {
+    used false
+  }
+
+  frames body:world
```

Comment:
Very well spotted, that you need this here!

But this probably requires `"..."`. But I'm not even sure. This is boosts "info" format. It is very compact and easy but not very well documented :(. It should definitely provide a yaml input. XML works nicely already but is not very human :).